src/sdk.js

```diff
@@ -104,7 +104,7 @@ class OpenForm {
     this.render();
   }
 
-  async render() {
+  render() {
```

Comment:
IMO this gave the impression that the rendering is awaitable, which it is not (but it doesn't hurt either)